compass/ocean/tests/isomip_plus/forward.py

```diff
@@ -145,6 +142,22 @@ def run(self):
         """
         Run this step of the test case
         """
+        config = self.config
+        resolution = self.resolution
+
+        dt_per_km = config.getfloat('isomip_plus', 'dt_per_km')
+        dt_btr_per_km = config.getfloat('isomip_plus', 'dt_btr_per_km')
+
+        # https://stackoverflow.com/a/1384565/7728169
+        # Note: this will drop any fractional seconds, which is usually okay
+        dt = time.strftime('%H:%M:%S', time.gmtime(dt_per_km * resolution))
+        btr_dt = time.strftime(
+            '%H:%M:%S', time.gmtime(dt_btr_per_km * resolution))
+
+        options = dict(config_dt="'{}'".format(dt),
+                       config_btr_dt="'{}'".format(btr_dt))
```

Comment:
Let's stick with f-strings:
```suggestion
        options = dict(config_dt=f"'{dt}'",
                       config_btr_dt=f"'{btr_dt}'")
```